docs/migrate_to_v2.md

```diff
@@ -28,33 +28,41 @@ const auth = function () {
 
 #### v2 Authentication: Credentials object
 
-```javascript
-{ 
+```typescript
+import { Credentials } from 'skyflow-node';
+
+const credentials: Credentials = { 
     apiKey: "<YOUR_SKYFLOW_API_KEY>"
-}
+};
```

Comment:
This is overkill. The section is describing the object itself not how to create one with types.